src/jsonschema.rs

```diff
@@ -18,6 +18,8 @@ enum Atom {
     String,
     Object,
     Array,
+#[serde(rename = "date-time")]
+    Datetime,
```

Comment:
It looks like these types should actually be `kebab-case` on the [container attributes](https://serde.rs/container-attrs.html#rename_all). I would then rename this `DateTime`.